src/uct/api/addr.h

```diff
@@ -7,6 +7,7 @@
 #ifndef UCT_API_ADDR_H_
 #define UCT_API_ADDR_H_
 
+#include "config.h"
```

Comment:
ok…we would have to come up with some nicer solution. Since no-body uses the address directly it can be moved to smsg transport and not been exposed publicly